redisinsight/api/src/modules/cloud/job/cloud-job.controller.ts

```diff
@@ -28,10 +34,20 @@ export class CloudJobController {
   })
   async createFreeDatabase(
     @RequestSessionMetadata() sessionMetadata,
-      @Body() dto: CreateCloudJobDto,
+      @Body() data: CreateCloudJobDto,
       @Query() utm: CloudRequestUtm,
   ): Promise<CloudJobInfo> {
-    return this.service.create(sessionMetadata, dto, utm);
+    const dto = plainToClass(CreateCloudJobDto, data);
```

Comment:
There is no need to validate something manually on controller layer. We already have data validated
see `@UsePipes(new ValidationPipe({ transform: true }))` and       `@Body() dto: CreateCloudJobDto,` does this automatically

Reply:
changed